src/test/java/nextstep/subway/acceptance/PathAcceptanceTest.java

```diff
@@ -0,0 +1,112 @@
+package nextstep.subway.acceptance;
+
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import nextstep.subway.applicaion.dto.PathResponse;
+import nextstep.subway.exception.CustomException;
+import nextstep.subway.exception.ErrorResponse;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+
+import static nextstep.subway.acceptance.LineSectionAcceptanceTest.createSectionCreateParams;
+import static nextstep.subway.acceptance.LineSteps.*;
+import static nextstep.subway.acceptance.StationSteps.지하철역_생성_요청;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("지하철 경로 검색")
+class PathAcceptanceTest extends AcceptanceTest {
+    private Long 교대역;
+    private Long 강남역;
+    private Long 양재역;
+    private Long 남부터미널역;
+    private Long 이호선;
+    private Long 신분당선;
+    private Long 삼호선;
+
+    /**
+     * 교대역    --- *2호선* (10) ---   강남역
+     *    |                           |
+     * *3호선* (2)                  *신분당선* (10)
+     *    |                           |
+     * 남부터미널역  --- *3호선* (3)---   양재
+     */
+    @BeforeEach
+    public void setUp() {
+        교대역 = 지하철역_생성_요청("교대역").jsonPath().getLong("id");
+        강남역 = 지하철역_생성_요청("강남역").jsonPath().getLong("id");
+        양재역 = 지하철역_생성_요청("양재역").jsonPath().getLong("id");
+        남부터미널역 = 지하철역_생성_요청("남부터미널역").jsonPath().getLong("id");
+
+        이호선 = 지하철_노선_생성_요청("2호선", "green").jsonPath().getLong("id");
+        신분당선 = 지하철_노선_생성_요청("신분당선", "red").jsonPath().getLong("id");
+        삼호선 = 지하철_노선_생성_요청("3호선", "orange").jsonPath().getLong("id");
+
+        지하철_노선에_지하철_구간_생성_요청(이호선, createSectionCreateParams(교대역, 강남역, 10));
+        지하철_노선에_지하철_구간_생성_요청(신분당선, createSectionCreateParams(강남역, 양재역, 10));
+        지하철_노선에_지하철_구간_생성_요청(삼호선, createSectionCreateParams(교대역, 남부터미널역, 2));
+        지하철_노선에_지하철_구간_생성_요청(삼호선, createSectionCreateParams(남부터미널역, 양재역, 3));
+    }
+
+    @Test
+    @DisplayName("최단 거리를 조회한다.")
+    void paths_When_두_역_사이의_최단_경로를_조회하면_Then_최단_경로를_조회한다() {
+        //when
+        ExtractableResponse<Response> response = 경로_조회_요청(교대역, 양재역);
+
+        //then
+        var pathResponse = response.as(PathResponse.class);
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+        assertThat(pathResponse.getStationResponse()).hasSize(3);
+        assertThat(pathResponse.getDistance()).isEqualTo(5);
```

Comment:
```suggestion
        //when
        var pathResponse = 경로_조회_요청(교대역, 양재역);

        //then
        assertThat(pathResponse.getStationResponse()).hasSize(3);
        assertThat(pathResponse.getDistance()).isEqualTo(5);
```

이렇게도 가능하지 않을까요~? 추가로 역 이름에 대한 검증도 할 것 같네요.

Reply:
[4a9f35e](https://github.com/next-step/atdd-subway-path/pull/526/commits/4a9f35eca9c1a033668eda4513d7ed4eb7fe5325)
역 이름 검증 추가했습니다! 🚀

또한 `경로_조회_요청`에서 바로 PathResponse를 받아오지 않고
예외 상황 Test case 대비하여 유연하여 사용할 수 있게 유지하였습니다!